packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -90,11 +92,13 @@ export const RTC_CONFIGURATION_FIREFOX = {iceServers: [], bundlePolicy: 'max-com
 export const RTC_CONFIGURATION = {iceServers: []};
 export const RESOURCE_ROOM = 'RESOURCE_ROOM';
 export const READY = 'ready';
+export const RECVONLY = 'RECVONLY';
 
 export const SIP_URI = 'SIP_URI';
 export const STOP = 'STOP';
 export const SHARE_PEER_CONNECTION_NAME = 'SharePeerConnection';
 export const SELF = 'SELF';
+export const SENDRECV = 'SENDRECV';
```

Comment:
Should we call this `SEND_RECEIVE` to be consistent with our own conventions? Or let this implementation detail remain so it's more obvious?

Reply:
same as above